app/_layout.tsx

```diff
@@ -26,42 +28,51 @@ function RootStack() {
     getPools();
   });
 
+  if (!privyAppId || !privyClientId) {
+    throw new Error('Privy App ID and Client ID are required');
+  }
```

Comment:
Maybe this could be moved into `constants/env`?

Reply:
done too